### PR TITLE
AFC-1369 - upgrade partner links 

### PR DESCRIPTION
**Pull request by iammiracle**

### CODE REVIEW

**Comment by FredrikArvas** [comment]
Hi again @iammiracle 
Could you please add the Jira Number on each commit and a short explanation of what you have changed.
@MickeHolm (who is our Configuration Mangager) is really keen that we all make a note in the commit-message of what we have done. The reason for this is that it makes it a lot easier to search for where a certain change was introduced in the code in the future when the change is stored in the commit-message as well as in the PR-message.
Thanks

**Comment by iammiracle** [comment]
Hi @FredrikArvas can you please help me with the Jira Number, where does this come from. What I have done is made sure once a contact card is open, the link method, runs and it shows on the tab

**Comment by FredrikArvas** [comment]
The Jira number is the same as the branch-number. 

Here are two examples: 

First an example from Daniel, where he have made a PR for two modules in one Jira
image

And here an example where I have added the information we want. 
(we have not been good in communicating what we want here...) 
So please start with this from now for all code that is unmerged.
image

@MickeHolm please confirm these instructions when you are back to work.
[Uploading Module-naming.pptx…]()

**Comment by iammiracle** [comment]
contact_links, AFC-1369 - upgrade partner links

@FredrikArvas I hope I am correct with this

**Comment by FredrikArvas** [comment]
Yes you are!
:-)

File: mobile_punch_clock/i18n/mobile_punch_clock.pot [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Translation of Odoo Server.
```
Comment (MickeHolm) [inline mobile_punch_clock/i18n/mobile_punch_clock.pot:1]: Why is this file added?
Please remove.

File: project_issue_sudo/i18n/project_issue_sudo.pot [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# Translation of Odoo Server.
```
Comment (MickeHolm) [inline project_issue_sudo/i18n/project_issue_sudo.pot:1]: Why is this file added?
Please remove.

File: contact_links/__manifest__.py [hunk 3]
```diff
@@ -15,20 +15,22 @@
...
-        'hr_360_view'
+        'hr_360_view',
+        'partner_view_360',
     ],
     'data': [
         'security/security.xml',
         'views/res_partner_view.xml',
         'security/ir.model.access.csv',
-        'data/server_action.xml',
+        # 'data/server_action.xml',
```
Comment (MickeHolm) [inline contact_links/__manifest__.py:33]: Why just comment out this?
Should it be removed? And if so, then the file data/server_action.xml should be removed as well.

**Review by MickeHolm — CHANGES_REQUESTED** [review]
Please update the commit messages to the format mentioned above.

File: contact_links/__manifest__.py (hunk 3 above)
Comment (iammiracle) [reply contact_links/__manifest__.py:33]: That was commented out because its not going to be needed, but I left this here, if it case it will be needed after testing what I did.

File: base_afdemo/data/calendar_event.csv [hunk 4]
```diff
@@ -1,285 +0,0 @@
-id;partner_ids/id;partner_ids/name;attendee_ids/id;attendee_ids/display_name;start;duration;meeting_status;location_id;location;name;description
```
Comment (MickeHolm) [inline base_afdemo/data/calendar_event.csv:1]: Why is this file removed?
What does it have to do with contact_links?

**Review by MickeHolm — CHANGES_REQUESTED** [review]
@iammiracle, Why is the file "ipf_ais_a/i18n/sv_SE.mo" included in this pull request?
And please fix the commit messages so that they conform with the format mentioned above:
contact_links, AFC-1369 - upgrade partner links and ...

File: base_afdemo/data/calendar_event.csv (hunk 4 above)
Comment (iammiracle) [reply base_afdemo/data/calendar_event.csv:1]: @MickeHolm this is what happened. The .mo and .po file extension was not in the .gitignore, so i added it, cleared the cache, then push. The base_afdemo/data/calendar_event.csv is inside the .gitignore file, so that is the reason. I did not add it there, you can see for yourself
